react/private/FieldLabel/FieldLabel.demo.js

```diff
@@ -9,6 +12,13 @@ export default [
           ...props,
           secondaryLabel: '(Max 30 characters)'
         })
+      },
+      {
+        label: 'Tertiary label',
+        transformProps: props => ({
+          ...props,
+          tertiaryLabel: <TextLink>Tertiary TextLink</TextLink>
```

Comment:
Could you make this a more realistic example, like the `secondaryLabel` above? Maybe 'Help'? That would probably fix the text wrapping issue, too.